core/src/main/java/kafka/server/share/ShareFetchUtils.java

```diff
@@ -46,59 +47,59 @@ public class ShareFetchUtils {
 
     // Process the replica manager fetch response to update share partitions and futures. We acquire the fetched data
     // from share partitions.
-    static CompletableFuture<Map<TopicIdPartition, ShareFetchResponseData.PartitionData>> processFetchResponse(
+    static Map<TopicIdPartition, ShareFetchResponseData.PartitionData> processFetchResponse(
```

Comment:
Should we remove futures in the comment above?

Reply:
Done.